code/controllers/subsystem/tts.dm

```diff
@@ -192,11 +200,21 @@ SUBSYSTEM_DEF(tts)
 					SHIFT_DATA_ARRAY(queued_tts_messages, tts_target, data)
 				continue
 			var/sound/audio_file = new(current_target.audio_file)
+			var/sound/audio_file_blips = new(current_target.audio_file_blips)
 			if(current_target.local)
-				SEND_SOUND(tts_target, audio_file)
+				var/use_blips
+				if(istype(current_target.target, /client)) // this can be a client sometimes instead of a mob for some fucking reason
```

Comment:
remove this comment

Reply:
also, you should tie `use_blips` to the tts_request instead, or the local request.
I don't think we should be doing unnecessary double processing by sending two HTTP requests when we only need to send one.
Consider refactoring the code a bit so that it only needs to send 1 HTTP request if it only needs to send 1.

Wasted processing is wasted processing. Even if we have the processing power to spare, other servers may not.

Reply:
@Watermelon914 MSO literally requested we send two requests